content/en/docs/event-sources/_index.md

```diff
@@ -3,12 +3,29 @@ title: Event Sources
 weight: 60
 ---
 
-Falco can consume events from a variety of different sources and apply rules to these events to detect abnormal behavior.
+Falco is able to consume streams of events and evaluate them agains a set of security rules to detect abnormal behavior. Events can be consumed by various event sources, which defined the origin, nature, and format of the streamed events.
 
-Falco natively supports the System Call event source (`syscall`) via the [drivers](./drivers). Since [Falco 0.31](../../blog/falco-0-31-0.md), Falco also supports additional event sources through the [Plugin System](../plugins):
+Falco natively supports the `syscall` event source, through which is able to consume events coming from the Linux Kernel by instrumenting it with the [drivers](./drivers). Since [Falco 0.31](/blog/falco-0-31-0.md) and the introduction of the [Plugin System](/docs/plugins), Falco can be configured to support additional event sources defined by plugins implementing [the event sourcing capability](/docs/plugins#event-sourcing-capability). Examples of event source defined by offically-supported are:
 
 * [Kubernetes Audit Events](./kubernetes-audit)
 * [AWS CloudTrail](./cloudtrail)
 * [Okta](./okta)
 
 In addition to these plugins hosted by the Falcosecurity organization, others have written third-party [plugins](https://github.com/falcosecurity/plugins#readme) that support additional event sources. Please refer to the [official Plugin Registry](https://github.com/falcosecurity/plugins/blob/master/registry.yaml) for the most up-to-date information regarding the Falco plugins acknowledged by the community.
+
+## Configuring Event Sources
+
+Historically, Falco supported consuming events coming from one event source only. This means that in order to consume events from more than one event source users needed to deploy many distinct instances of Falco, each one configured with a different source. The only exception was the legacy implementation of the [Kubernetes Audit Events](./kubernetes-audit) which allowed receiving events from the `k8s_audit` event sources in parallel with the `syscall` one, however it was non-standardized and substituted in favor of a [plugin-based porting](https://github.com/falcosecurity/plugins/blob/master/plugins/k8saudit/README.md) starting from [Falco 0.32.0](/blog/falco-0-32-0.md). Since [version 0.33.0](/blog/falco-0-33-0.md), Falco introduced a standard **support for consuming events from multiple event sources** running in parallel within the same Falco instance. You can check the design principles and rationale of this feature at [falcosecurity/falco/issues/2074](https://github.com/falcosecurity/falco/issues/2074).
```

Comment:
```suggestion
Historically, Falco supported consuming events coming from one event source only. This means that to consume events from more than one event source, users needed to deploy many instances of Falco, each configured with a different source. The only exception was the legacy implementation of the [Kubernetes Audit Events](./kubernetes-audit) which allowed receiving events from the `k8s_audit` event sources in parallel with the `syscall` one. However, it was non-standardized and has been substituted in favor of a [plugin-based porting](https://github.com/falcosecurity/plugins/blob/master/plugins/k8saudit/README.md) starting from [Falco 0.32.0](/blog/falco-0-32-0.md). Since [version 0.33.0](/blog/falco-0-33-0.md), Falco introduced a standard **support for consuming events from multiple event sources** running in parallel within the same Falco instance. You can check this feature's design principles and rationale at [falcosecurity/falco/issues/2074](https://github.com/falcosecurity/falco/issues/2074).
```